src/java/picard/analysis/artifacts/ContextAccumulator.java

```diff
@@ -48,9 +42,12 @@ public void countRecord(final String refContext, final char calledBase, final SA
      */
     public ListMap<Transition, DetailPair> calculateMetrics(final String sampleAlias, final String library) {
         final ListMap<Transition, DetailPair> detailMetricsMap = new ListMap<>();
-        for (final Transition altTransition : Transition.altValues()) {
-            final Transition refTransition = altTransition.matchingRef();
-            for (final String context : new TreeSet<>(this.artifactMap.get(altTransition).keySet())) {
+        for (final String context : new TreeSet<>(artifactMap.keySet())) {
+
+            final char refBase = context.charAt(context.length() / 2);
```

Comment:
this part of the code doesn't happen that much though...it's not like the accumulators, right?